<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserDataWidget</class>
 <widget class="QGroupBox" name="CUserDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Data:</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>User Data</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Prefix:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="prefix"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="avatar">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Icon Aspect Ratio:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QDoubleSpinBox" name="avatarAspectRatio"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="setAvatarBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="EmbySync.qrc">
             <normaloff>:/resources/missingAvatar.png</normaloff>:/resources/missingAvatar.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Connect ID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="connectID"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Connect ID Type:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="connectIDType">
         <item>
          <property name="text">
           <string>Guest</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LinkedUser</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Creation Date (UTC):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QDateTimeEdit" name="creationDate">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="displayFormat">
            <string>M/d/yyyy hh:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setCreatedDateToNow">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="EmbySync.qrc">
             <normaloff>:/resources/clock.png</normaloff>:/resources/clock.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Last Login Date (UTC):</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QDateTimeEdit" name="lastLoginDate">
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="displayFormat">
            <string>M/d/yyyy hh:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setLastLoginDateToNow">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="EmbySync.qrc">
             <normaloff>:/resources/clock.png</normaloff>:/resources/clock.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Last Activity Date (UTC):</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QDateTimeEdit" name="lastActivityDate">
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="displayFormat">
            <string>M/d/yyyy hh:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setLastActivityDateToNow">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="EmbySync.qrc">
             <normaloff>:/resources/clock.png</normaloff>:/resources/clock.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="enableAutoLogin">
         <property name="text">
          <string>Enable Auto Login?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Configuration Data</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Audio Language Preference:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="audioLanguagePreference"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="playDefaultAudioTrack">
         <property name="text">
          <string>Play Default Audio track?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Subtitle Language Preference</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="subtitleLanguagePreference"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="displayMissingEpisodes">
         <property name="text">
          <string>Display Missing Episodes?</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Subtitle Mode:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="subtitleMode">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OnlyForced</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Smart</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="enableLocalPassword">
         <property name="text">
          <string>Enable Local Password?</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="hidePlayedInLatest">
         <property name="text">
          <string>Hide Played In Latest?</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="rememberAudioSelections">
         <property name="text">
          <string>Remember Audio Selections?</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="rememberSubtitleSelections">
         <property name="text">
          <string>Remember Subtitle Selections?</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="enableNextEpisodeAutoPlay">
         <property name="text">
          <string>Enable Next Episode Auto Play?</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Resume Rewind Seconds:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="resumeRewindSeconds">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Intro Skip Mode:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QComboBox" name="introSkipMode">
         <item>
          <property name="text">
           <string>ShowButton</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AutoSkip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Configuration Data</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Ordered Views:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QListWidget" name="orderedViews">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Latest Items Excludes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="latestItemsExcludes">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>My Media Excludes:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QListWidget" name="myMediaExcludes">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="process">
       <property name="text">
        <string>&amp;Process</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/upload.png</normaloff>:/resources/upload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="text">
        <string>Apply From Here</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/arrowright.png</normaloff>:/resources/arrowright.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name</tabstop>
  <tabstop>prefix</tabstop>
  <tabstop>avatarAspectRatio</tabstop>
  <tabstop>setAvatarBtn</tabstop>
  <tabstop>connectID</tabstop>
  <tabstop>connectIDType</tabstop>
  <tabstop>creationDate</tabstop>
  <tabstop>setCreatedDateToNow</tabstop>
  <tabstop>lastLoginDate</tabstop>
  <tabstop>setLastLoginDateToNow</tabstop>
  <tabstop>lastActivityDate</tabstop>
  <tabstop>setLastActivityDateToNow</tabstop>
  <tabstop>enableAutoLogin</tabstop>
  <tabstop>process</tabstop>
  <tabstop>apply</tabstop>
  <tabstop>audioLanguagePreference</tabstop>
  <tabstop>playDefaultAudioTrack</tabstop>
  <tabstop>subtitleLanguagePreference</tabstop>
  <tabstop>displayMissingEpisodes</tabstop>
  <tabstop>subtitleMode</tabstop>
  <tabstop>enableLocalPassword</tabstop>
  <tabstop>hidePlayedInLatest</tabstop>
  <tabstop>rememberAudioSelections</tabstop>
  <tabstop>rememberSubtitleSelections</tabstop>
  <tabstop>enableNextEpisodeAutoPlay</tabstop>
  <tabstop>resumeRewindSeconds</tabstop>
  <tabstop>introSkipMode</tabstop>
  <tabstop>orderedViews</tabstop>
  <tabstop>latestItemsExcludes</tabstop>
  <tabstop>myMediaExcludes</tabstop>
 </tabstops>
 <resources>
  <include location="EmbySync.qrc"/>
 </resources>
 <connections/>
</ui>
